users/views.py

```diff
@@ -9,7 +9,58 @@
 from users.validators import UserValidator
 from my_settings      import SECRET_KEY, ALGORITHM
 
+class SignupView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            name          = data['name']
+            phone         = data['phone']
+            email         = data['email']
+            date_of_birth = data['date_of_birth']
+            password      = data['password']
+            gender        = data['gender']
+
+            V = UserValidator()
+            V.validate_name(name)
+            V.validate_phone(phone)
+            V.validate_email(email)
+            V.validate_date_of_birth(date_of_birth)
+            V.validate_password(password)
+            V.validate_gender(gender)
+
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
+
+            if User.objects.filter(email=email).exists():
+                return JsonResponse({'MESSAGE' : 'EMAIL_ALREADY_EXIST'}, status = 400)
+
+            if User.objects.filter(phone=phone).exists():
+                return JsonResponse({'MESSAGE' : 'PHONE_ALREADY_EXIST'}, status = 400)
+
+            user = User.objects.create(
+                name          = name,
+                phone         = phone,
+                email         = email,
+                date_of_birth = date_of_birth,
+                password      = hashed_password,
+                gender        = gender
+            )
+
+            info = {'id' : user.id}
+            
+            access_token = jwt.encode(info, SECRET_KEY, ALGORITHM)
```

Comment:
최대한 읽어야 되는 line도 적으면 좋겠죠?

```suggestion
            access_token = jwt.encode({'id' : user.id}, SECRET_KEY, ALGORITHM)
```

Reply:
훨씬 더 깔끔한거 같아요, 바로 수정하겠습니다!